src/Bot.ts

```diff
@@ -1,44 +1,69 @@
 "use strict";
 
+import { calculateBotPick } from "mtgdraftbots";
+
 import { Card } from "./Cards";
 
+const basicOracleIds = [
+	"56719f6a-1a6c-4c0a-8d21-18f7d7350b68",
+	"b2c6aa39-2d2a-459c-a555-fb48ba993373",
+	"bc71ebf6-2056-41f7-be35-b2e5c34afa99",
+	"b34bb2dc-c1af-4d77-b0b3-a0fb342a5fc6",
+	"a3fb7228-e76b-4e96-a40e-20b5fed75685",
+];
+
 export default class Bot {
 	name: string;
 	id: string;
-	cards: Card[] = []; // For debugging mostly.
-	pickedColors: { [color: string]: number } = { W: 0, U: 0, R: 0, B: 0, G: 0 };
+	oracleIds: string[] = [];
```

Comment:
```suggestion
	oracleIds: string[] = [...basicOracleIds];
```
Basics IDs have to be included, right?